front/app/modules/free/customizable_navbar/admin/components/messages.ts

```diff
@@ -15,76 +15,76 @@ export default defineMessages({
     defaultMessage: 'Navigation',
   },
   navbarItemTitle: {
-    id: 'app.modules.commercial.customizable_navbar.admin.components.NavbarTitleField.navbarItemTitle',
+    id: 'app.modules.free.customizable_navbar.admin.components.NavbarTitleField.navbarItemTitle',
```

Comment:
Are we sure that Crowdin will find these translations? If not, maybe we should leave `commercial` in copy paths?

Reply:
What will happen if you change the id is that these will become new translations. The trick I once used was also replacing the ids/keys in the .json files, but to be confirmed how this works.

So you can try replacing them in the JSON files (search and replace), but they have already been deleted since you committed to GitHub, which triggered the script. I guess the safest is to keep the old IDs and revert the removal of existing translations until we know we can use this.

Reply:
I'm 99% sure that if the default string is the same, Crowdin will see it as a new translation (based on a new key) but just use all translated versions from the old key. We have "regular duplicate detection" on in Crowdin which works like this: 
`You can save time by translating all duplicates with the same translation and hiding these instances from translators. This may, however, affect accuracy.
Regular duplicates detection considers only source texts`

I'm not sure how this works in terms of the timing, but we could run a test

Reply:
Yes, you can make a new branch, change 1 string and see if it works as you say.

Reply:
Indeed, they're [marked with grey](https://crowdin.com/translate/citizenlabdotcocitizenlab/7364/en-frbe?filter=basic&value=0#q=customizable_navbar) as already translated
![image](https://user-images.githubusercontent.com/91883434/196399586-45b922dc-2a83-41bb-b663-92ee71395b43.png)